src/docs/development/tools/sdk/release-notes/supported-platforms.md

```diff
@@ -23,13 +23,15 @@ but that the development team does not directly test or support.
 
 |Platform|Version              |
 |-------|----------------------|
+|Android|Windows SDK 30        |
 |Android|Android SDK 29        |
 |Android|Android SDK 28        |
 |Android|Android SDK 27        |
 |Android|Android SDK 26        |
 |Android|Android SDK 25        |
 |Android|Android SDK 24        |
 |Android|Android SDK 23        |
+|Android|Android SDK 22        |
 |Android|Android SDK 21        |
 |Android|Android SDK 19        |
```

Comment:
We may want to call out here or below that SDK 20 is covered by testing 19 for Flutter's purposes.